plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/ListContainersParams.java

```diff
@@ -0,0 +1,175 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.docker.client.params;
+
+import org.eclipse.che.plugin.docker.client.json.Filters;
+
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Arguments holder for {@link org.eclipse.che.plugin.docker.client.DockerConnector#listContainers(ListContainersParams)}.
+ *
+ * @author Alexander Andrienko
+ */
+public class ListContainersParams {
+    private Boolean all;
+    private Integer limit;
+    private String  since;
+    private String  before;
+    private Boolean size;
+    private Filters filters;
+
+    private ListContainersParams() {
+    }
+
+    /**
+     * Creates and returns arguments holder.
+     */
+    public static ListContainersParams from() {
```

Comment:
API is really new and as far as i know the version with it hasn't been released yet, so we can consider a different name for all the methods like this, lets say it is `.create`. WDYT @mmorhun, @garagatyi ?